addons/core/tests/dummy/app/templates/smoke-test.hbs

```diff
@@ -9,7 +9,7 @@
   Ember can start up and basic features work as intended.
 </Hds::Text::Body>
 
-<Hds::Text::Display @tag='h2'>{{t
+<Hds::Text::Display @tag='h2' class='remove-header-margin'>{{t
```

Comment:
Here is an example of using this new class to remove margin during those one-offs that we do not want margins for our header.

Reply:
Does this have the right specificity? I think this would get overridden by the HDS margin overrides and wouldn't do anything

Reply:
Fixed.